.eslintrc

```diff
@@ -97,6 +97,17 @@
       "rules": {
         "oppia/e2e-action": "error",
       }
+    },
+    {
+      // We only run the protractor practices checks on protractor end-to-end test files.
+      "files": [
+        "core/tests/protractor/*.js",
```

Comment:
This insure that protractor-practices check only runs for protractor-related files.